pom.xml

```diff
@@ -1083,55 +680,6 @@ under the License.
 			</build>
 		</profile>
 
-		<profile>
-			<!-- Kept for backwards compatiblity, the doc buildbot expects
-				 this profile to exist.-->
-			<id>aggregate-scaladoc</id>
-		</profile>
-
-
-		<profile>
-			<!-- used for SNAPSHOT and regular releases -->
-			<id>docs-and-source</id>
```

Comment:
Do not intend to publish javadocs? (And if so, how shall we integrate them into the docs?)